src/Framework/Http/ConnectServer/Client/Exceptions/RequestException.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Give\Framework\Http\ConnectServer\Client\Exceptions;
+
+use Give\Framework\Exceptions\Primitives\RuntimeException;
+
+/**
+ * @unreleased
+ */
+class RequestException extends RuntimeException
```

Comment:
Should this extend our Loggable exception base class?

Reply:
Resolved in 9c9d78e710cace0a740a6669d8927cc4347a633f